core/src/main/java/org/jdbi/v3/core/locator/ClasspathSqlLocator.java

```diff
@@ -75,7 +79,11 @@ private static String resourcePathFor(String fullyQualifiedName) {
     }
 
     private static String findSqlOnClasspath(ClassLoader classLoader, String path) {
-        return CACHE.computeIfAbsent(path, p -> readResource(classLoader, p));
+        try {
+            return CACHE.get(path, () -> readResource(classLoader, path));
```

Comment:
Unfortunately that's only a method on `LoadingCache` not `Cache` itself -- I can't fathom why!  But we should probably fix the code to include the ClassLoader anyway, so I'll do that now...